app/apps/server/communication/rest.js

```diff
@@ -249,11 +249,18 @@ export class AppsRestApi {
 			},
 		});
 
-		this.api.addRoute('games', { authRequired: false }, {
+		this.api.addRoute('externalComponents', { authRequired: false }, {
 			get() {
-				const games = orchestrator.getEnabledGames();
+				const registeredExternalComponents = orchestrator.getRegisteredExternalComponents();
+				const externalComponents = [];
 
-				return API.v1.success({ games: Array.from(games.values()) });
+				registeredExternalComponents.forEach((appExternalComponents) => {
```

Comment:
Move this logic inside of the Engine code base and into a new method that only returns an array of external components which are enabled and ready for usage.

Reply:
Resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/150/commits/34b5a9b95ec5acff2befaa781567abedf57c88a3